packages/marketplace/contracts/RoyaltiesRegistry.sol

```diff
@@ -203,7 +200,9 @@ contract RoyaltiesRegistry is OwnableUpgradeable, IRoyaltiesProvider {
     /// @return royalties 2981 royalty array
     function _getRoyaltiesEIP2981(address token, uint256 tokenId) internal view returns (Part[] memory royalties) {
         try IERC2981(token).royaltyInfo(tokenId, WEIGHT_VALUE) returns (address receiver, uint256 royaltyAmount) {
-            try IERC165Upgradeable(token).supportsInterface(INTERFACE_ID_GET_RECIPIENTS) returns (bool result) {
+            try IERC165Upgradeable(token).supportsInterface(IMultiRoyaltyRecipients.getRecipients.selector) returns (
```

Comment:
I'm not sure which one is wrong, but it's not implemented the same way on the asset contract
https://github.com/thesandboxgame/sandbox-smart-contracts/blob/master/packages/dependency-royalty-management/contracts/MultiRoyaltyDistributor.sol#L43

Reply:
I verified that: INTERFACE_ID_GET_RECIPIENTS == IMultiRoyaltyRecipients.getRecipients.selector 
On the other hand type(IMultiRoyaltyRecipients).interfaceId may include IERC165 too.